ui/js/directives/treeherder/clonejobs.js

```diff
@@ -818,8 +818,16 @@ treeherder.directive('thCloneJobs', [
 
         var isOnScreen = function(el){
             var viewport = {};
-            viewport.top = $(window).scrollTop() + $("#global-navbar-container").height();
-            viewport.bottom = viewport.top + $(window).height() - $("#info-panel").height();
+            viewport.top = $(window).scrollTop() + $("#global-navbar-container").height() + 20;
```

Comment:
This and the "- 20" down below are to make sure if a job partially overlaps the header or info-panel elements, the job fully scrolls into view.